src/keylessBackup/WalletSecurityPrimer.tsx

```diff
@@ -34,7 +38,14 @@ function WalletSecurityPrimer({ route }: Props) {
         testID="WalletSecurityPrimer/GetStarted"
         onPress={function () {
           ValoraAnalytics.track(KeylessBackupEvents.wallet_security_primer_get_started)
-          navigate(Screens.SetUpKeylessBackup)
+          dispatch(
+            keylessBackupStarted({
+              keylessBackupFlow: KeylessBackupFlow.Setup,
+            })
+          )
```

Comment:
I believe this was moved to SignInWithEmail as part of #4811

Reply:
Fixed in 2573f3e705e03eb70ad51204714fda2b1ba589d0!